ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/logging.j2

```diff
@@ -28,6 +28,12 @@ no logging monitor
 {%     elif logging.monitor is arista.avd.defined %}
 logging monitor {{ logging.monitor }}
 {%     endif %}
+{%     if logging.event.storm_control.discards.enable is arista.avd.defined(true) %}
+logging event storm-control discards global
+{%     endif %}
+{%     if logging.event.storm_control.discards.interval is arista.avd.defined %}
+logging event storm-control discards interval {{ logging.event.storm_control.discards.interval }}
+{%     endif %}
```

Comment:
Should this be nested under the enabled check? Or is there a valid use case for setting the interval globally but not enabling globally?
If the interval depends on the `enable` key, we should make the `enable` key _required_  in the schema. (Add `required: true`)

Reply:
@ClausHolbechArista 

I just oriented to ptp under ethernet_interface, but I understand what you mean.
It´s your decision.
```
{%     if ethernet_interface.ptp.enable is arista.avd.defined(true) %}
   ptp enable
{%     endif %}

{%     if ethernet_interface.ptp.delay_req is arista.avd.defined %}
   ptp delay-req interval {{ ethernet_interface.ptp.delay_req }}
{%     endif %}
```
```
        ptp:
          type: dict
          keys:
            enable:
              type: bool

            delay_req:
              type: int
              convert_types:
                - str
```

Reply:
I verified on EOS, and indeed they are separate, so you should not nest them.